tests/microos/one_line_checks.pm

```diff
@@ -14,7 +14,8 @@ use utils;
 
 sub run_rcshell_checks {
     # Check that system is using UTC timezone
-    assert_script_run 'date +"%Z" | grep -x UTC';
+    my $timezone = get_var('FIRST_BOOT_CONFIG', '') =~ /cloud-init/ ? 'CEST'  : 'UTC';
```

Comment:
```suggestion
    my $timezone = get_var('FIRST_BOOT_CONFIG') =~ /cloud-init/ ? 'CEST' : 'UTC';
```

Reply:
The variable is not always defined. In such a case, the comparison would yield a warning

Reply:
anyway, the `space` more after CEST, probably is tidy alert.